test/functionalTest/cases/1316_create_subscription/post_subs_expr_q_not_equal_enum_number.test

```diff
@@ -0,0 +1,236 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+POST Subscriptions V2 q expression (not enum string)
+
+--SHELL-INIT--
+dbInit CB
+# FIXME P10: replace with just 'brokerStart CB' once cache part gets implemented
+brokerStart CB 0 IPv4 -noCache
+accumulatorStart
+
+--SHELL--
+
+#
+# 01. Create subscription with q: avg!=23,47
+# 02. Create entity with avg=23 (not notif)
+# 03. Update entity to avg=35 (notif)
+# 04. Update entity to avg=47 (not notif)
+# 05. Update entity to avg=93 (notif)
+# 06. Dump accumulator (2 notifications)
+#
+
+echo "01. Create subscription with q: avg!=23,47"
+echo "=========================================="
+payload='
+{
+    "subject": {
+        "entities": [
+            {
+                "id": "E1",
+                "type": "T"
+            }
+        ],
+        "condition": {
+            "attributes": [ ],
+            "expression": {
+               "q": "avg!=23,47"
+            }
+         }
+    },
+    "notification": {
+        "callback": "http://localhost:'$LISTENER_PORT'/notify",
+        "attributes": [ ]
+    },
+    "expires": "2050-04-05T14:00:00.00Z"
+}
+'
+orionCurl --url /v2/subscriptions --payload "$payload" --json
+echo
+echo
+
+
+echo "02. Create entity with avg=23 (not notif)"
+echo "========================================="
+payload='{
+  "id": "E1",
+  "type": "T",
+  "avg": 23
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "03. Update entity to avg=35 (notif)"
+echo "==================================="
+payload='{
+  "avg": 35
+}'
+orionCurl --url /v2/entities/E1 --payload "$payload" --json -X PATCH
+echo
+echo
+
+
+echo "04. Update entity to avg=47 (not notif)"
+echo "======================================="
+payload='{
+  "avg": 47
+}'
+orionCurl --url /v2/entities/E1 --payload "$payload" --json -X PATCH
+echo
+echo
+
+
+echo "05. Update entity to avg=93 (notif)"
+echo "==================================="
+payload='{
+  "avg": 93
+}'
+orionCurl --url /v2/entities/E1 --payload "$payload" --json -X PATCH
+echo
+echo
+
+
```

Comment:
You only use integers in this test.
Yet, the test is called 'not enum string' ...
It should work with strings as well, right?